### PR TITLE
Cleanup Graph API tests even more. . . Almost There. 

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

**Comment by fppt** [comment]
please retest this please - ai.grakn.test.engine.GraknEngineFailoverIT.whenSubmittingTasksToOneEngineAndRandomlyKillingTheOthers_TheyComplete

**Comment by fppt** [comment]
please retest this please - ai.grakn.test.engine.GraknEngineFailoverIT.whenSubmittingTasksToOneEngineAndRandomlyKillingTheOthers_TheyComplete

File: grakn-graph/src/main/java/ai/grakn/graph/internal/ConceptImpl.java [hunk 1]
```diff
@@ -144,13 +144,6 @@ private boolean updateAllowed(Schema.ConceptProperty key, String value) {
      * Deletes the node and adds it neighbours for validation
      */
     void deleteNode(){
-        // tracking
-        getVertex().edges(Direction.BOTH).
-                forEachRemaining(
-                        e -> {
-                            graknGraph.getConceptLog().trackConceptForValidation(getGraknGraph().getElementFactory().buildConcept(e.inVertex()));
-                            graknGraph.getConceptLog().trackConceptForValidation(getGraknGraph().getElementFactory().buildConcept(e.outVertex()));}
-                );
```
Comment (aelred) [inline grakn-graph/src/main/java/ai/grakn/graph/internal/ConceptImpl.java]: Is this definitely not necessary anymore?

Comment (fppt) [inline grakn-graph/src/main/java/ai/grakn/graph/internal/ConceptImpl.java]: Nope. It was too low level. It was leading to redundant validations.

**Pull request merged**